tests/unit/api.py

```diff
@@ -664,8 +672,8 @@ def test_headnode_delete_hnic_success(self, db):
         api.headnode_create('hn-0', 'anvil-nextgen', 'base-headnode')
         api.headnode_create_hnic('hn-0', 'hn-0-eth0')
         api.headnode_delete_hnic('hn-0', 'hn-0-eth0')
-        api._assert_absent(db, model.Hnic, 'hn-0-eth0')
-        hn = api._must_find(db, model.Headnode, 'hn-0')
+        api._assert_absent(model.Hnic, 'hn-0-eth0')
+        hn = api._must_find(model.Headnode, 'hn-0')
```

Comment:
Hm.  I wonder if this test is missing a line at the end... it's weird that it ends by setting a variable